### ISSUE TITLE
Errors occurring in bitcoin28.0

### ISSUE THREAD

**Issue by wakiyamap** [trezor/blockbook#1138]
in journal.

```blockbook[842240]: E1009 01:37:47.251446  842240 blockbook.go:383] rpc: json: cannot unmarshal array into Go struct field .result.warnings of type string Retrying...```

This can be fixed by adding the following to bitcoin.conf:
`deprecatedrpc=warnings`

example
```
# generated from additional_params
deprecatedrpc=warnings
deprecatedrpc=estimatefee
```

ref.
https://github.com/lightningnetwork/lnd/issues/9163
https://github.com/bitcoin/bitcoin/pull/29845

**Comment by martinboehm** [comment]
Thank you!

**Comment by martinboehm** [comment]
Fixed by #1140
